File: javascript/gpt/p15.js
// p15: function returns value if in bounds, else returns 0
const readline = require("readline");

function getValue(idx) {
  const id_sequence = [11, 22, 33, 44];
  if (idx >= 0 && idx < id_sequence.length) return id_sequence[idx];
  return 0;
}

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});
rl.question("Index: ", (ans) => {
  const idx = parseInt(ans.trim(), 10);
  if (Number.isNaN(idx)) {
    console.log(0);
  } else {
    console.log(getValue(idx));
  }
  rl.close();
});

File: python/gpt/p15.py
# p15: function returns value at index if in bounds, otherwise returns 0


def get_value(idx):
    id_sequence = [11, 22, 33, 44]
    if 0 <= idx < len(id_sequence):
        return id_sequence[idx]
    return 0


def main():
    try:
        idx = int(input("Index: "))
    except Exception:
        print(0)
        return 0
    print(get_value(idx))
    return 0


if __name__ == "__main__":
    main()
